src/App.js

```diff
@@ -67,12 +68,23 @@ class Root extends React.Component {
           this.props.history.push('/wallet_type/');
         }
       });
+
+      IPC_RENDERER.on("ledger:manyTokenSignatureValid", (event, arg) => {
```

Comment:
Use single quotes instead of double quotes

Reply:
Done